assets/javascripts/kitten/components/lists/icon-list/components/item.js

```diff
@@ -0,0 +1,52 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { pxToRem } from '../../../../helpers/utils/typography'
+import TYPOGRAPHY from '../../../../constants/typography-config'
+import COLORS from '../../../../constants/colors-config'
+import { ScreenConfig } from '../../../../constants/screen-config'
+import { Icon } from './icon'
+
+export class Item extends PureComponent {
+  static propTypes = {
+    icon: PropTypes.node,
+    iconProps: PropTypes.object,
+  }
+
+  static defaultProps = {
+    icon: null,
+    iconProps: {},
+  }
+
+  render() {
+    const { icon, iconProps, children, ...other } = this.props
+
+    return (
+      <StyledItem {...other}>
+        <Icon {...iconProps}>{icon}</Icon>
+
+        <StyledContent>{children}</StyledContent>
+      </StyledItem>
+    )
+  }
+}
+
+const StyledItem = styled.li`
+  display: flex;
+  margin ${pxToRem(40)} 0;
+`
+
+const StyledContent = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
```

Comment:
Sans l'indication `flex: 0 1 auto;`, est-ce que ça marche bien avec IE11 ?

(Voir Flexbugs : https://github.com/philipwalton/flexbugs#1-minimum-content-sizing-of-flex-items-not-honored)

Reply:
Bien vu. J'ai du mettre un `flex: 1;` pour IE 11.